src/Donations/resources/admin/components/DonationDetailsPage/Tabs/Overview/DonationReceipt/index.tsx

```diff
@@ -36,9 +37,11 @@ export default function DonationReceipt({ donation }: { donation: Donation }) {
           </div>
         </div>
 
-        <nav className={styles.actions} aria-label={__('Receipt actions', 'give')}>
-          <ReceiptActions />
-        </nav>
+        {donation?.status === 'publish' && (
```

Comment:
Shouldn't we also check for `give_subscription` here? That's the "completed" status for renewals.